src/Microsoft.CSharp/Microsoft.CSharp.sln

```diff
@@ -2,17 +2,17 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.CSharp.Tests", "tests\Microsoft.CSharp.Tests.csproj", "{82B54697-0251-47A1-8546-FC507D0F3B08}"
```

Comment:
Did you run this update on your Mac? If so we should probably normalize the slashes in our task to avoid these noop updates. This will definitely cause VS to rewrite all these solution files.